code/modules/mob/living/carbon/xenomorph/hive_status.dm

```diff
@@ -979,6 +979,7 @@
 	need_round_end_check = TRUE
 
 	var/list/defectors = list()
+	var/list/personal_allies = list()
```

Comment:
Please explicit list type. In general, are allies specifically humans or any carbons or even any mob? Different code features point to all 3 - i'm arguing internally this should be /mob/living everywhere even if you decide to only allow adding human allies

Reply:
yeah it meant to be human only, can you explain why it should be /mob/living everywhere though?

Reply:
Well my reasoning is that CORRUPTED_ALLY is a status_flags so relevant to /mob,  and is_ally will get called for xenos too. I don't see a reason to fence the whole thing to only work with humans rather than living or carbon. You can always just only allow humans to be added when selecting allies